src/test/java/bio/terra/app/controller/SnapshotValidationTest.java

```diff
@@ -68,14 +68,14 @@
 class SnapshotValidationTest {
 
   @Autowired private MockMvc mvc;
-  @MockBean private JobService jobService;
-  @MockBean private SnapshotService snapshotService;
-  @MockBean private IamService iamService;
-  @MockBean private IngestRequestValidator ingestRequestValidator;
-  @MockBean private FileService fileService;
-  @MockBean private AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
-  @MockBean private SnapshotBuilderService snapshotBuilderService;
-  @SpyBean private ApplicationConfiguration applicationConfiguration;
+  @MockitoBean private JobService jobService;
+  @MockitoBean private SnapshotService snapshotService;
+  @MockitoBean private IamService iamService;
+  @MockitoBean private IngestRequestValidator ingestRequestValidator;
+  @MockitoBean private FileService fileService;
+  @MockitoBean private AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
+  @MockitoBean private SnapshotBuilderService snapshotBuilderService;
+  @MockitoSpyBean private ApplicationConfiguration applicationConfiguration;
```

Comment:
```suggestion
  @MockitoBean private ApplicationConfiguration applicationConfiguration;
```
This appeared to work on my local test. (Thanks @pshapiro4broad  for the help!)
It looks like `@SpyBean` will generate a Bean if it doesn't already exist, but `@MockitoSpyBean` won't create a Bean.